packages/model-serving/src/components/deploymentWizard/fields/ProjectSection.tsx

```diff
@@ -37,9 +36,13 @@ const ProjectSection: React.FC<ProjectSectionType> = ({
   projectName,
   setProjectName,
 }) => {
+  const projectLabel = 'This is the OpenShift AI project where the model will be deployed.';
```

Comment:
We can't hardcode OpenShift AI since this will show up in ODH. I think you can use https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/.env#L7C1-L7C5